2022/09/07 15:58:54 [error] 7#7: *6 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 192.168.32.1, server: parser.test.ru, request: "GET / HTTP/2.0", upstream: "fastcgi://192.168.32.5:9000", host: "parser.localhost"
2022/09/07 15:59:55 [error] 8#8: *3 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 192.168.32.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 192.168.32.1, server: parser.test.ru, request: "GET / HTTP/2.0", upstream: "fastcgi://192.168.32.5:9000", host: "parser.localhost"
2022/09/07 16:00:25 [error] 8#8: *3 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 192.168.32.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 192.168.32.1, server: parser.test.ru, request: "GET / HTTP/2.0", upstream: "fastcgi://192.168.32.5:9000", host: "parser.localhost"
2022/09/07 16:04:19 [crit] 8#8: *6 SSL_do_handshake() failed (SSL: error:140944E7:SSL routines:ssl3_read_bytes:reason(1255):SSL alert number 255) while SSL handshaking, client: 192.168.32.1, server: 0.0.0.0:443
2022/09/07 16:04:19 [crit] 8#8: *7 SSL_do_handshake() failed (SSL: error:140944E7:SSL routines:ssl3_read_bytes:reason(1255):SSL alert number 255) while SSL handshaking, client: 192.168.32.1, server: 0.0.0.0:443
2022/09/10 07:39:25 [error] 7#7: *4 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 192.168.32.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 192.168.32.1, server: parser.test.ru, request: "GET / HTTP/2.0", upstream: "fastcgi://192.168.32.5:9000", host: "parser.localhost"
2022/09/10 07:46:29 [error] 7#7: *4 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 192.168.32.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 192.168.32.1, server: parser.test.ru, request: "GET /api/v1/import HTTP/2.0", upstream: "fastcgi://192.168.32.5:9000", host: "parser.localhost"
2022/09/10 08:33:15 [error] 7#7: *31 upstream timed out (110: Connection timed out) while reading response header from upstream, client: 192.168.32.1, server: parser.test.ru, request: "GET /api/v1/import HTTP/2.0", upstream: "fastcgi://192.168.32.5:9000", host: "parser.localhost"
2022/09/10 09:30:20 [error] 7#7: *47 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 192.168.32.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 192.168.32.1, server: parser.test.ru, request: "GET /api/v1/import HTTP/2.0", upstream: "fastcgi://192.168.32.5:9000", host: "parser.localhost"
2022/09/10 10:55:59 [error] 7#7: *100 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 192.168.32.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 192.168.32.1, server: parser.test.ru, request: "GET /api/v1/import HTTP/2.0", upstream: "fastcgi://192.168.32.5:9000", host: "parser.localhost"
2022/09/10 10:56:01 [error] 7#7: *100 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 192.168.32.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 192.168.32.1, server: parser.test.ru, request: "GET /_ignition/health-check HTTP/2.0", upstream: "fastcgi://192.168.32.5:9000", host: "parser.localhost", referrer: "https://parser.localhost/api/v1/import"
2022/09/10 10:56:10 [error] 7#7: *100 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 192.168.32.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 192.168.32.1, server: parser.test.ru, request: "GET /api/v1/import HTTP/2.0", upstream: "fastcgi://192.168.32.5:9000", host: "parser.localhost"
2022/09/10 10:59:34 [crit] 7#7: *105 SSL_do_handshake() failed (SSL: error:140944E7:SSL routines:ssl3_read_bytes:reason(1255):SSL alert number 255) while SSL handshaking, client: 192.168.32.1, server: 0.0.0.0:443
2022/09/10 10:59:38 [error] 7#7: *107 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 192.168.32.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 192.168.32.1, server: parser.test.ru, request: "GET /api/v1/import HTTP/2.0", upstream: "fastcgi://192.168.32.5:9000", host: "parser.localhost"
